queue_services/entity-filer/devops/vaults.gcp.env

```diff
@@ -31,4 +31,8 @@ NATS_CLIENT_NAME="op://nats/$APP_ENV/nats-filer/NATS_CLIENT_NAME"
 NATS_FILER_SUBJECT="op://nats/$APP_ENV/nats-filer/NATS_FILER_SUBJECT"
 NATS_ENTITY_EVENT_SUBJECT="op://nats/$APP_ENV/nats-filer/NATS_ENTITY_EVENT_SUBJECT"
 NATS_EMAILER_SUBJECT="op://nats/$APP_ENV/nats-emailer/NATS_EMAILER_SUBJECT"
+AUDIENCE="op://gcp-queue/$APP_ENV/payment/AUDIENCE"
```

Comment:
I'm wondering about this /payment directory in 1Pass. Maybe we need one for each queue service? Especially for the TOPIC_NAME below.

Reply:
yeah, we should probably add sections in 1pass for each queue i think.